Update smp-supported-platforms.md (#5530)
harness/developer-hub: docs/self-managed-enterprise-edition/shared/smp-supported-platforms.md

### Terms of support

> The support policy is 12 months of support and followed up with 6 months of limited support for critical security fixes only.

Reviewer: ```suggestion
The support policy is 12 months of full support, followed by 6 months of limited support for critical security fixes only.
```
